static/js/components/PersonalTab_test.js

```diff
@@ -24,6 +24,7 @@ describe("PersonalTab", () => {
             ui={{selectedProgram: selectedProgram}}
             dispatch={store.dispatch}
             uneditedProfile={USER_PROFILE_RESPONSE}
+            profile={USER_PROFILE_RESPONSE}
```

Comment:
Sorry this took forever to answer! Ran into a bunch of issues with `yarn` that I couldn't get past (in order to get the tests running).

Basically, the phone number input adds a `'+'` to the input if it is empty (on page-load), which fires off it's `onChange` handler (and then calls the `updateProfile` function). Before we were not passing a `profile` to `PersonalTab`, which was fine, because we aren't testing here any functionality relating to the profile editing, but now this behavior of the empty phone number input meant that, in the personal tab test, that `onChange` handler was firing, and throwing an error because it couldn't find the `updateProfile` prop. Passing a `profile` prop ensures that the phone number input has a value, so it doesn't try to do this thing.